### PR TITLE
Modified the checkout scripts to parse arguments

### PR DESCRIPTION
**Pull request by mcvsubbu**
Unified the two arg parsers to have the same type of arguments.

## Upgrade Notes
Does this PR prevent a zero down-time upgrade? (Assume upgrade order: Controller, Broker, Server, Minion)
* [ ] Yes (Please label as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR fix a zero-downtime upgrade introduced earlier?
* [ ] Yes (Please label this as **<code>backward-incompat</code>**, and complete the section below on Release Notes)

Does this PR otherwise need attention when creating release notes? Things to consider:
- New configuration options
- Deprecation of configurations
- Signature changes to public methods/interfaces
- New plugins added or old plugins removed
* [ ] Yes (Please label this PR as **<code>release-notes</code>** and complete the section on Release Notes)

### CODE REVIEW

**Review by GSharayu — APPROVED** [review]

**Review by mqliang — APPROVED** [review]

**Comment by mqliang** [comment]
Please update the README.md. Can be a followup PR.

**Review by mqliang — APPROVED** [review]

**Comment by mcvsubbu** [comment]
@mqliang I updated the README and also pulled common function out to a different file.

File: compatibility-verifier/utils.inc [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#compute absolute path from a relative path
```
Comment (jackjlli) [inline compatibility-verifier/utils.inc:1]: nit: put one space between `#` and the actual comment

File: compatibility-verifier/utils.inc [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+function absPath() {
+  local relPath=$1
+  if [[ ! "$relPath" == /* ]]; then
+    #relative path
```
Comment (jackjlli) [inline compatibility-verifier/utils.inc:5]: Same here

**Review by jackjlli — APPROVED** [review]
Minor but LGTM

**Pull request merged**